src/sentry/lang/javascript/plugin.py

```diff
@@ -15,7 +14,7 @@ def preprocess_event(data):
     translate_exception(data)
     fix_culprit(data)
     if data.get('platform') == 'javascript':
-        inject_device_data(data)
+        normalize_user_agent(data)
```

Comment:
Is this still necessary if we already perform `normalize_user_agent` in the `event_manager`?

Reply:
it will be removed there once we validated that it works. For now we start with a small sample rate of events that get it as part of normalize since we don't know the performance impact of this.